### ISSUE TITLE
command 'extension.snipsnap_activate' not found

### ISSUE THREAD

**Issue by theadameden** [snipsnapdev/snipsnap-vscode#17]
Installed the extension on Popos 20.04, vs code installed via flatpak.

Opening command pallet > Snipsnap: Fetch the snippets results in;
Command 'Snipsnap: Fetch the snippets' resulted in an error (command 'extension.snipsnap_activate' not found)

Let me know of other information you may require to help diagnose this one.

**Comment by mmoga** [comment]
I'm experiencing the same issue. VS Code v1.14.2

**Comment by BolotskyDev** [comment]
Currently [there is only one activation event that triggers this ext:](https://github.com/snipsnapdev/snipsnap-vscode/blob/master/package.json#L19) workspace should contain `package.json`. Can you confirm that your workspace indeed contains `package.json`, but still nothing is working? 🤔

**Comment by mmoga** [comment]
@BolotskyDev Thanks for getting back to me. I created a new project (and it has a `package.json`) and it's working as expected. Sorry for the waste of time. 😞

**Comment by theadameden** [comment]
@BolotskyDev Yes, in fact it contains multiples. Maybe that's the problem, creating a project with just on package.json in the root works fine. A mono-repo with many does not. Even a mono-repo with one package.json that it not in the root does not work.

**Comment by paragbaxi** [comment]
building microservices app which requires a package.json file per service. would be good if this could support multiple package.json. i would expect that snipsnap would use the package.json file from the closest parent directory relative to the file that it was triggered on.

**Comment by akd-io** [comment]
Fetching is also not working for me in a workspace with a single folder containing the project including package.json.
Would indeed be very nice with multiple package.json support
